index.html

```diff
@@ -506,6 +512,12 @@ <h2>Profiles</h2>
 gaming persona, or home life persona.
       </p>
 
+      <p class="note">
+It is possible to have a <a>profile</a>, such as a business persona,
+that contains multiple <a>credentials</a> about different <a>subjects</a>
+that are often, but not required to be, related.
```

Comment:
"that are not required to be related". (strike "often").

Reply:
Fixed.